backend/routers/userRoutes.js

```diff
@@ -43,8 +43,12 @@ router.patch(
     authController.updatePassword
 );
 router.get("/me", authController.protect, userController.getMe);
-
-router.use(authController.protect);
+// update
+router.patch(
+    "/me/update",
+    authController.protect,
+    userController.updateProfile
+);
 
```

Comment:
Can you add one more endpoint we required, just return true or false (check if username is duplicate, for use after user type in frontend.